soh/src/overlays/actors/ovl_En_Cow/z_en_cow.c

```diff
@@ -209,12 +216,129 @@ void func_809DF730(EnCow* this, GlobalContext* globalCtx) {
     }
 }
 
+struct CowInfo {
+    int cowId;
+    RandomizerCheck randomizerCheck;
+};
+
+struct CowInfo EnCow_GetCowInfo(EnCow* this, GlobalContext* globalCtx) {
+    s32 uniqueCoords = this->actor.world.pos.x + this->actor.world.pos.z;
+    struct CowInfo cowInfo;
+
+    switch (globalCtx->sceneNum) {
+        case SCENE_SOUKO: // Lon Lon Tower
+            switch (uniqueCoords) {
+                // Two cases here cause this cow is moved in randomizer
```

Comment:
we shouldn't need 2 cases here because we shouldn't be calling this outside of rando

Reply:
For cleanliness this method is also used before the cows are moved to identify which should be moved, so I need both cases here or a separate function that would look mostly the same, opted for this.

Reply:
but if we're not randomized we don't move cows, so why does this need to be called at all in that case?

Reply:
oh, i think i get it, you're using this to identify the cow before moving it, and then again after moving it

it'd be nice if the comment in code made that more clear, "cause this cow is moved in randomizer" doesn't really get that point across